### PR TITLE
Update it.json

### PR DESCRIPTION
**Pull request by enricomg**
new patch

### CODE REVIEW

File: js/web/_i18n/it.json [hunk 1]
```diff
@@ -54,12 +54,12 @@
 		"Boxes.CityMap.TitleSend": "Invia dati",
 		"Boxes.CityMap.WholeArea": "L'intera area:",
 		"Boxes.CityMap.YourCity": "La tua città",
-		"Boxes.EventChests.Cost": "Costs", //Todo: Translate
-		"Boxes.EventChests.CostPerDailyPrize": "Cost/__dailyprize__", //Todo: Translate
```
Comment (Th3C0D3R) [inline js/web/_i18n/it.json]: Please dont translate those words with __xxx___

Comment (enricomg) [reply js/web/_i18n/it.json]: ok

**Comment by enricomg** [comment]
I close the pull, do I make it new?

**Comment by Th3C0D3R** [comment]
jeah please make a new one i will close this

**Pull request closed without merging**